.github/workflows/test.yml

```diff
@@ -63,35 +63,6 @@ jobs:
       - run: yarn add --dev @storybook/addon-actions@7 @storybook/addon-interactions@7 @storybook/api@7 @storybook/manager-api@7 @storybook/preview-api@7 @storybook/react@7 @storybook/react-webpack5@7 storybook@7 @storybook/test@7
       - run: yarn build && npx -p happo.io happo-ci-github-actions
 
-  storybook6:
```

Comment:
I'm okay with dropping support for storybook 6. But I also wanted to provide some data about this version.

I think the latest v6 was 6.5.16 published January 26, 2023. npm says that this version has been downloaded 110,593 times in the last 7 days. 

For comparison, 8.6.14 is the latest v8 and was published about a month ago and has 1,201,893 downloads in the last 7 days. Recent v9 releases have about 100k downloads.